tsdb/engine/tsm1/engine_test.go

```diff
@@ -609,6 +609,65 @@ func TestEngine_DeleteSeries(t *testing.T) {
 
 }
 
+func TestEngine_LastModified(t *testing.T) {
+	// Generate temporary file.
+	f, _ := ioutil.TempFile("", "tsm")
```

Comment:
Are you using the generated filename as a directory? Why not just use `iotuil.TempDir`?